recOrder/compute/qlipp_compute.py

```diff
@@ -155,28 +151,20 @@ def reconstruct_qlipp_birefringence(stokes, recon):
     ----------
     stokes                  : np.ndarray or zarr array
                               stokes array generated by reconstruct_qlipp_stokes
+                              dimensions: (C, Y, X, Z) or (C, Y, X)
 
     recon                   : waveorder.waveorder_microscopy object
                               initialized by initialize_reconstructor
 
     Returns
     -------
         recon_data          : np.ndarray
-                              volume of shape (Z, C, Y, X) containing reconstructed birefringence data.
+                              volume of shape (C, Z, Y, X) or (C, Y, X) containing reconstructed birefringence data.
     """
 
-    if len(stokes.shape) == 4:
-        slices = stokes.shape[0]
-        recon_data = np.zeros([slices, 4, stokes.shape[-2], stokes.shape[-1]])
-    elif len(stokes.shape) == 3:
-        slices = 1
-        recon_data = np.zeros([1, 4, stokes.shape[-2], stokes.shape[-1]])
-
-    for z in range(slices):
-        recon_data[z, :, :, :] = recon.Polarization_recon(stokes[z] if slices != 1 else stokes)
-
+    birefringence = recon.Polarization_recon(stokes)
 
-    return np.transpose(recon_data, (1,0,2,3))
+    return np.transpose(birefringence, (0, 3, 1, 2)) if len(birefringence.shape) == 4 else birefringence
```

Comment:
Just for documentation -- I understand that a lot of the changes I see here are due to waveorder reconstruction needing dimensions like (C, Y, X, Z).